src/dealWizard/stages/proposalLeadStage/proposalLeadStage.ts

```diff
@@ -0,0 +1,35 @@
+import { autoinject } from "aurelia-framework";
+import { RouteConfig } from "aurelia-router";
+import { IBaseWizardStage } from "../../dealWizard.types";
+import { WizardService, IWizard } from "../../../services/WizardService";
+
+@autoinject
+export class ProposalLeadStage implements IBaseWizardStage {
+  public wizardManager: any;
+  public wizard: IWizard;
+  public errors: Record<string, string> = {};
+
+  constructor(public wizardService: WizardService) {}
+
+  activate(_params: unknown, routeConfig: RouteConfig): void {
+    this.wizardManager = routeConfig.settings.wizardManager;
+  }
+
+  attached(): void {
+    this.wizard = this.wizardService.getWizard(this.wizardManager);
+  }
+
+  validateInputs(): boolean {
+    this.errors = {};
```

Comment:
can a [validation controller](https://aurelia.io/docs/plugins/validation#validation-controller) be used for validating form fields?

Reply:
Thanks for the hint, I will look into that